Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -296,6 +304,8 @@ void ConsoleLineEdit::ResetHistoryIndex()
 
 Lines CConsoleSCB::s_pendingLines;
 
+QMenu* optionsMenu;
+
```

Comment:
I think this global variable was added on accident, please remove.

Reply:
Thanks, removed.